front/src/components/AdminPanel/Panel/Categories/Categories.js

```diff
@@ -1,8 +1,163 @@
+                                onChange={(event) => setCategoryGender(event.target.value)}>
+                                <option value="woman" key="woman"> Kobiety </option>
+                                <option value="man" key="man"> Mężczyźni </option>
+                            </select>{" "}
+                            <br />
+                            <Button
+                                type="submit"
+                                placeholder="Dodaj kategorię"
+                                className="btn btn-primary btn-lg"
+                            />
+                        </form>
+                    </div>
+                    <div className = "category-tile-div">
+                        <p className="categories-header"> Wybierz kategorię, aby dodać odległość </p>
+                        <table className="table table-hover" style={{marginLeft: "2%"}, {width:"90%"}}>
+                            <tbody>
+                            {
+                                Object.keys(categoriesNames).map(function(element) {
+                                    return <tr className={chosenCategory == categoriesNames[element]['id'] && "activeRow"}> <td> <input type="radio" name="catName" value={categoriesNames[element]['id']} onChange={(event) => { setChosenCategory(event.target.value); setChosenDistance(false); }}/> {categoriesNames[element]['name']}  </td></tr>
+                                })
+                            }
+                            </tbody>
+                        <br/>
+                        <Button
+                                type="submit"
+                                placeholder="Usuń kategorię"
+                                className="btn btn-primary btn-lg"
+                            />
+                        </table>
+                    </div>
+                </div>
+                <div className="category-right-div">
+                {
+                    chosenCategory && <div className = "category-tile-div">
+                        <p className="categories-header"> "{categoriesNames[chosenCategory-1]['name']}" - nowa odległość </p> 
+                        <form>
+                            <input
+                                type="text"
+                                placeholder="Nazwa odległości"
+                                className="form-control"
+                                value={distance}
+                                onChange={(event) => setDistance(event.target.value)}
+                            />{" "}
+                            <br />
+                            <input
+                                type="number"
+                                placeholder="Ilość serii"
+                                className="form-control"
+                                value={seriesAmount}
+                                onChange={(event) => setSeriesAmount(event.target.value)}
+                            />{" "}
```

Comment:
Add constraint for this input, so that number of series can only be positive integer (currently it accepts negative values as well)